library/std/src/io/prelude.rs

```diff
@@ -1,4 +1,4 @@
-//! The I/O Prelude
+//! # The I/O Prelude
```

Comment:
Why is this change being made?

Reply:
Because it's basically the title of the page, so it looks better as a heading. The way it is now looks kind of funny:

![image](https://user-images.githubusercontent.com/37223377/103247085-812b2d80-491a-11eb-9317-13c27fc6a9bc.png)

It's also consistent with the docs at the `std` root, which use `# The Rust Standard Library`. Though if we make this change, we should probably also make `std::prelude`'s title a heading (currently it's not).

Reply:
Yes so, I would rather change this in Rustdoc than make every single heading have a `#`. The convention is generally to not include this, and so by doing it on one or two pages, it's gonna be weird for those and not all other code in existence.

Reply:
> Yes so, I would rather change this in Rustdoc than make every single heading have a `#`.

I'm not sure what you mean by "change this in Rustdoc". Can you explain a bit?

> The convention is generally to not include this, and so by doing it on one or two pages, it's gonna be weird for those and not all other code in existence.

Other pages seem to usually use lowercase text with a period at the end. (E.g., [`std::alloc`](https://doc.rust-lang.org/nightly/std/alloc/index.html).) Maybe it would be best to change this to use that style?

I feel like perhaps `std::prelude` should use a heading though, to be consistent with `# The Rust Standard Library` and `# The Rust Core Library`.

Reply:
> I'm not sure what you mean by "change this in Rustdoc". Can you explain a bit?

If you want the summary line to be larger, I would change Rustdoc to produce this output, rather than changing the contents of the doc string.

> Other pages seem to usually use lowercase text with a period at the end.

Yep, that seems good to me, see https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#summary-sentence

> to be consistent with # The Rust Standard Library and # The Rust Core Library.

Arguably those are inconsistent with the rest of the docs.

Reply:
> If you want the summary line to be larger, I would change Rustdoc to produce this output, rather than changing the contents of the doc string.

I don't want the summary line to be larger; I'm not sure what I said that made you think that :)

Also, changing it to be a heading shouldn't change the contents of the summary line at all.